wiki/Installation/zh.md

```diff
@@ -1,186 +1,73 @@
-
-#### osu!stream
-
-##### 如何安装
-> 警告：osu!stream *不是* osu!。
-这是一个和 osu! 很相似的游戏，但它有自己的特色和排名。
-
-##### 通过 App Store 安装（推荐）
-> 确保你的设备已经连接上 Internet（使用无线局域网或蜂窝移动网络）。
-
-- 在你的设备上打开 App Store。
-- 搜索 osu!stream。
-- 安装 osu!stream。
-  - 如果提示你输入 Apple ID 和密码，请这样做。
-- 下载并自动安装完成后即可运行。
-
-##### 通过 USB 安装
-> 你的计算机必须已经安装 [iTunes](https://www.apple.com/itunes/)。
-
-- 打开[这个链接](https://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8)并下载它
-  - 可能需要你输入 Apple ID 和密码。
-- 将设备与计算机连接。
-- 使用 iTunes 传输 osu!stream 文件。
-- 断开设备与计算机的连接，现在应该可以运行 osu!stream 了。
-
-##### 添加谱面
-> 确保你的设备已经连接上 Internet（使用无线局域网或蜂窝移动网络）。
-
-- 打开 osu!stream。
-- 点击 osu! 图标，你应该会前往到主菜单。
-- 点击 "Store" 条。
-  - 你也可以点击 "Play" 然后点击 "Download more songs!"。
-- 选择你想要的谱面。
-  - 建议你先下载免费的谱面，免费的谱面有 "FREE" 标签。
-
-## 常见问题和故障排除
-
-### osu!
-- 我尝试运行 osu!，但是出现了 "Initialization Error"！
-  - 确保你已经安装了 [.NET FrameWork 4.0](https://www.microsoft.com/zh-cn/download/details.aspx?id=17718)。
-- 为什么我无法下载谱面？
-  - 首先，你需要注册（如果你还没有注册）并登录。注册是免费的，只需要几步，只有登录后才能下载谱面、在游戏内查看在线排名或与其它游戏玩家聊天，因此你没有理由不注册！
-- 我不能玩下载好的谱面！
-  - 这是因为你的谱面文件不完整或已损坏，这可能是因为网络原因/你取消了下载/谱面的制作者上传了损坏的谱面。你可以试着重新下载谱面或搜索相同歌曲名的其它谱面。
-   - osu! 将会把损坏的谱面放置在 "Songs/Failed" 目录下。
-- 发生了其它错误？
-  - 如果你遇到了一个没有在上面列出的错误，你可以前往[问题反馈板块](https://osu.ppy.sh/community/forums/5)并发表帖子(可能会有其它人同时或已经碰到相关的问题，所以建议你先搜索相关的帖子)。只要清楚地说出你的问题，并提供一个证据来表明这个问题是确实存在的（截图/视频/能重现该问题的操作步骤），一定会有人来帮助你的！
-
-### osu!droid
-
-- 哪些 Android 版本可以运行它？
-  - 1.6 以上的全部版本。
-- 将会支持故事板或视频吗？
-  - 不会。
-- 你希望添加新的功能么？
-  - 将你想要的新功能在这个[请求帖子](https://osu.ppy.sh/community/forums/topics/62313)中告诉我们。
-
-## 外部链接
-
-- peppy's FAQ [1](https://osu.ppy.sh/p/faq)
-- peppy's download page [2](https://osu.ppy.sh/p/download)
-- peppy's osu! (OS X) testing release [3](https://osu.ppy.sh/community/forums/posts/909738)
-- An unofficial guide on [installing osu! on macOS](/wiki/Installation/macOS)
-- KcLKcL's Solution: Running Osu! In Ubuntu or Xubuntu [5](https://osu.ppy.sh/forum/viewtopic.php?f=2&t=14614)
-- doomed151's General Information and Download (osu!droid) [6](https://osu.ppy.sh/community/forums/topics/62680)
-  - Pesets's osu!droid Changelog [7](https://osu.ppy.sh/community/forums/topics/62315)
-  - osu!droid Google Play page [8](https://play.google.com/store/apps/details?id=ru.nsu.ccfit.zuev.osu)
-- Guy-kun's T-Aiko! for Android (Market Release!) [9](https://osu.ppy.sh/community/forums/topics/58640)
-  - T-Aiko Google Play page [10](https://play.google.com/store/apps/details?id=guy.taiko&feature=search_result)
-- peppy's osu!stream official page [11](http://osustream.com/)
-  - osu!stream App Store page [12](https://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8)
+该页面会简要地告诉你怎么让 osu! 在 Windows 上运行。[osu!academy](/wiki/osu!academy) 制作了一个[视频教程](https://www.youtube.com/watch?list=PLmWVQsxi34bMYwAawZtzuptfMmszUa_tl&v=cz522ZAs5aQ)【译注：需要科学上网】来解释怎么在 Windows 上面安装 osu!。
+
+
+## 最低系统要求
+
+- .NET framework 4+（安装程序应该会提示你下载，如果没有，你可以在[这里](https://www.microsoft.com/en-us/download/details.aspx?id=48130)找到。
```

Comment:
```suggestion
- .NET framework 4+（安装程序应该会提示你下载，如果没有，你可以在[这里](https://www.microsoft.com/zh-cn/download/details.aspx?id=48130)找到。
```

You can link to the Chinese version of the page, if that works.

Reply:
Fixed.